release/release.sh

```diff
@@ -93,19 +117,38 @@ if ! command -v hub > /dev/null; then
   exit 1
 fi
 
-banner "Starting release for ${PROJECT} (${CLONE_URL})"
-hub clone "${PROJECT}" "${REPO_DIR}"  # May force login to GH at this point
+banner "Starting release for ${PROJECT_ARG} (${CLONE_URL})"
+hub clone "${PROJECT_ARG}" "${REPO_DIR}"  # May force login to GH at this point
 cd "${REPO_DIR}"
 
-# Figures out the most recent tagged version, and computes the next version.
-readonly TAG="$(git describe --tags --abbrev=0 origin/master)"
-readonly CUR_TAG="$(test -n "${TAG}" && echo "${TAG}" || echo "v0.0.0")"
-readonly NEW_RELEASE="$(perl -pe 's/v0.(\d+).0/"v0.${\($1+1)}"/e' <<<"${CUR_TAG}")"
-readonly NEW_TAG="${NEW_RELEASE}.0"
-readonly NEW_BRANCH="${NEW_RELEASE}.x"
+NEW_VERSION=""
+if [[ -n "${VERSION_ARG}" ]]; then
+  NEW_VERSION="${VERSION_ARG}"
+  echo "New version: ${NEW_VERSION}"
+else
+  # Figures out the most recent tagged version, and computes the next version.
+  readonly TAG="$(git describe --tags --abbrev=0 origin/master)"
+  readonly CUR_TAG="$(test -n "${TAG}" && echo "${TAG}" || echo "v0.0.0")"
+  readonly CUR_VERSION="${CUR_TAG#v}"
+  NEW_VERSION="$(perl -pe 's/(\d+).(\d+).(\d+)/"$1.${\($2+1)}.$3"/e' <<<"${CUR_VERSION}")"
+  echo "Current tag: ${CUR_TAG}"
+  echo "New version: ${NEW_VERSION}"
```

Comment:
Perhaps report tag+tag, or version+version, but not tag+version?

Reply:
Good suggestion. I changed it so the banner now prints something like `"release info for ${CUR_TAG} -> ${NEW_TAG}"`, both tags